department-of-veterans-affairs/va.gov-cms
# VACMS-3182: Paragraph Remove Button

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3182

## Testing done


## Screenshots


## QA steps

- Login to preview environment
- Navigate to the following pages and confirm that the **Remove** button is no longer visible within the **Include Alert** section and that you can successfully **Save**.
  - [ ] `/node/add/checklist`
  - [ ] `/node/add/faq_multiple_q_a`
  - [ ] `/node/add/support_resources_detail_page`
  - [ ] `/node/add/media_list_images`
  - [ ] `/node/add/media_list_videos`
  - [ ] `/node/add/q_a`
  - [ ] `/node/add/step_by_step`

- Navigate to existing content types to confirm that the *Remove** button is no longer visible within the **Include Alert** section.
  - [ ] `checklist`
  - [ ] `faq_multiple_q_a`
  - [ ] `support_resources_detail_page`
  - [ ] `media_list_images`
  - [ ] `media_list_videos`
  - [ ] `q_a`
  - [ ] `step_by_step`

## Definition of Done

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
